autoconnect/autoconnect-ws/src/error.rs

```diff
@@ -61,14 +61,13 @@ impl ReportableError for WSError {
     fn is_sentry_event(&self) -> bool {
         match &self.kind {
             WSErrorKind::SM(e) => e.is_sentry_event(),
-            e => !matches!(e, WSErrorKind::Json(_) | WSErrorKind::SessionClosed(_)),
+            _ => false,
```

Comment:
weren't we going with https://github.com/mozilla-services/autopush-rs/pull/478#discussion_r1372239508?